### DISCUSSION TITLE
Managing collection of products

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore.Commerce#458] [Q&A] [answered]
Hi,
If I wanted to manage some collection of products, should I use product list?
If so, how it can be done?

![](https://commerce.orchardcore.net/en/latest/assets/images/create-webshop/step-4/product-list.png)

**Comment by sarahelsaig** [comment]
Could you give more details about what you mean by "collection of products"? Do you want to group the products by categories, or something else?

**Reply by DotCat1985** [reply]
I'd like to group products by some parameters like category, materials, etc.
Also, I want to include some products in a fashion collection.

**Reply by sarahelsaig** [reply, marked as answer]
You can categorize content items using taxonomies, see [this tutorial](https://www.youtube.com/watch?v=0pmrmmqF8-4) and the OC docs on [taxonomies](https://docs.orchardcore.net/en/latest/reference/modules/Taxonomies/). Then, if you add a service which implements [`IProductListFilterProvider`](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/src/Modules/OrchardCore.Commerce/Services/ProductListTitleFilterProvider.cs) you will be able select which products to show in `ProductListPart`. Unfortunately we don't have a tutorial on that process yet (see #459) but you can check out the already existing [`ProductListTitleFilterProvider`](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/36676621498f4286b1ece84f0dfe6127a726f1fb/src/Modules/OrchardCore.Commerce/Services/ProductListTitleFilterProvider.cs) for inspiration.
